### PR TITLE
*: add performance improvement of system tables in release notes v8.0

### PR DESCRIPTION
**Pull request by Frank945946**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### 增加如下内容
- 优化了 INFORMATION_SCHEMA.TABLES，INFORMATION_SCHEMA.STATISTICS，INFORMATION_SCHEMA.KEY_COLUMN_USAGE，INFORMATION_SCHEMA.REFERENTIAL_CONSTRAINTS 的这 4 个系统表的查询性能，相比之前版本，性能最高有 100 倍的提升。

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-8.0.0.md [hunk 1]
```diff
@@ -392,6 +392,7 @@ TiDB 版本：8.0.0
     - 支持 MemoryTracker 追踪 `IndexLookup` 算子的内存使用情况 [#45901](https://github.com/pingcap/tidb/issues/45901) @[solotzg](https://github.com/solotzg)
     - 支持 MemoryTracker 追踪 `MemTableReaderExec` 算子的内存使用情况 [#51456](https://github.com/pingcap/tidb/issues/51456) @[wshwsh12](https://github.com/wshwsh12)
     - 支持从 PD 批量加载 Region，加快在对大表进行查询时，从 KV Range 到 Regions 的转换过程 [#51326](https://github.com/pingcap/tidb/issues/51326) @[SeaRise](https://github.com/SeaRise)
+    - 优化了 INFORMATION_SCHEMA.TABLES，INFORMATION_SCHEMA.STATISTICS，INFORMATION_SCHEMA.KEY_COLUMN_USAGE，INFORMATION_SCHEMA.REFERENTIAL_CONSTRAINTS 的这 4 个系统表的查询性能，相比之前版本，性能最高有 100 倍的提升。
```
Comment (Frank945946) [inline releases/release-8.0.0.md:395]:
```suggestion
    - 优化了 INFORMATION_SCHEMA.TABLES，INFORMATION_SCHEMA.STATISTICS，INFORMATION_SCHEMA.KEY_COLUMN_USAGE，INFORMATION_SCHEMA.REFERENTIAL_CONSTRAINTS 的这 4 个系统表的查询性能，相比之前版本，性能最高有 100 倍的提升。[#50305](https://github.com/pingcap/tidb/issues/50305) @[ywqzzy](https://github.com/ywqzzy)
```

**Review by ywqzzy — APPROVED** [review]

File: releases/release-8.0.0.md [hunk 2]
```diff
@@ -392,6 +392,7 @@ TiDB 版本：8.0.0
     - 支持 MemoryTracker 追踪 `IndexLookup` 算子的内存使用情况 [#45901](https://github.com/pingcap/tidb/issues/45901) @[solotzg](https://github.com/solotzg)
     - 支持 MemoryTracker 追踪 `MemTableReaderExec` 算子的内存使用情况 [#51456](https://github.com/pingcap/tidb/issues/51456) @[wshwsh12](https://github.com/wshwsh12)
     - 支持从 PD 批量加载 Region，加快在对大表进行查询时，从 KV Range 到 Regions 的转换过程 [#51326](https://github.com/pingcap/tidb/issues/51326) @[SeaRise](https://github.com/SeaRise)
+    - 优化了 INFORMATION_SCHEMA.TABLES，INFORMATION_SCHEMA.STATISTICS，INFORMATION_SCHEMA.KEY_COLUMN_USAGE，INFORMATION_SCHEMA.REFERENTIAL_CONSTRAINTS 的这 4 个系统表的查询性能，相比之前版本，性能最高有 100 倍的提升。[#50305](https://github.com/pingcap/tidb/issues/50305) @[ywqzzy](https://github.com/ywqzzy)
```
Comment (hfxsd) [inline releases/release-8.0.0.md:395]:
```suggestion
    - 优化系统表 `INFORMATION_SCHEMA.TABLES`、`INFORMATION_SCHEMA.STATISTICS`、`INFORMATION_SCHEMA.KEY_COLUMN_USAGE`、`INFORMATION_SCHEMA.REFERENTIAL_CONSTRAINTS` 的查询性能。相比之前版本，性能最高提升可达 100 倍 [#50305](https://github.com/pingcap/tidb/issues/50305) @[ywqzzy](https://github.com/ywqzzy)
```

**Review by hfxsd — APPROVED** [review]

File: releases/release-8.0.0.md [hunk 3]
```diff
@@ -392,6 +392,7 @@ TiDB 版本：8.0.0
     - 支持 MemoryTracker 追踪 `IndexLookup` 算子的内存使用情况 [#45901](https://github.com/pingcap/tidb/issues/45901) @[solotzg](https://github.com/solotzg)
     - 支持 MemoryTracker 追踪 `MemTableReaderExec` 算子的内存使用情况 [#51456](https://github.com/pingcap/tidb/issues/51456) @[wshwsh12](https://github.com/wshwsh12)
     - 支持从 PD 批量加载 Region，加快在对大表进行查询时，从 KV Range 到 Regions 的转换过程 [#51326](https://github.com/pingcap/tidb/issues/51326) @[SeaRise](https://github.com/SeaRise)
+    - 优化系统表 `INFORMATION_SCHEMA.TABLES`、`INFORMATION_SCHEMA.STATISTICS`、`INFORMATION_SCHEMA.KEY_COLUMN_USAGE`、`INFORMATION_SCHEMA.REFERENTIAL_CONSTRAINTS` 的查询性能。相比之前版本，性能最高提升可达 100 倍 [#50305](https://github.com/pingcap/tidb/issues/50305) @[ywqzzy](https://github.com/ywqzzy)
```
Comment (hfxsd) [inline releases/release-8.0.0.md:395]:
```suggestion
    - 优化系统表 `INFORMATION_SCHEMA.TABLES`、`INFORMATION_SCHEMA.STATISTICS`、`INFORMATION_SCHEMA.KEY_COLUMN_USAGE`、`INFORMATION_SCHEMA.REFERENTIAL_CONSTRAINTS` 的查询性能。相比之前版本，性能提升最高可达 100 倍 [#50305](https://github.com/pingcap/tidb/issues/50305) @[ywqzzy](https://github.com/ywqzzy)
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**